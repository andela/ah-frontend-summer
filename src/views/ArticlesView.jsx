import React from 'react';
import ArticleListContainer from "../containers/Articles/ArticleListContainer";


const ArticlesView = () => (
    <div>
        <ArticleListContainer title="Articles" paginating />
    </div>
);

export default ArticlesView;
