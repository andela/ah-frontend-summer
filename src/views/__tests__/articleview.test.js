import React from 'react';
import { shallow } from "enzyme";

import ArticlesView from '../ArticlesView';

describe('render Article View Container', () => {
    const wrapper = shallow(<ArticlesView />);
    it("renders rate component correctly", () => {
        expect(wrapper).toMatchSnapshot();
    });
});
