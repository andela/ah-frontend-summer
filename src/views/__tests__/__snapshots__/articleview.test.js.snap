// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Article View Container renders rate component correctly 1`] = `
<div>
  <Connect(ArticleListContainer)
    paginating={true}
    title="Articles"
  />
</div>
`;
